stdlib/public/core/FloatingPoint.swift

```diff
@@ -328,7 +328,14 @@ public protocol FloatingPoint: SignedNumeric, Strideable, Hashable
   /// `infinity` is greater than this value.
   static var greatestFiniteMagnitude: Self { get }
 
-  /// The mathematical constant pi.
+  /// The mathematical constant π, approximately equal to 3.14159.
```

Comment:
Let's put the wiki link in the summary: `The [mathematical constant π][wiki], ...` and remove it below.

Reply:
Makes sense, wasn't sure if it was appropriate (or even supported) there.